playground/vitestSetup.ts

```diff
@@ -243,23 +255,12 @@ export async function startDefaultServe(): Promise<void> {
       watcher = rollupOutput as RollupWatcher
       await notifyRebuildComplete(watcher)
     }
-    viteTestUrl = await startStaticServer(resolvedConfig, config)
+    viteTestUrl = await startStaticServer(rootConfig)
```

Comment:
Umm... It's very complicated here.

- `config`: config file near the *.spec.ts (non-resolved)
- `testConfig`: `mergeConfig(options, config || {})` (non-resolved)
- `rootConfig`: `mergeConfig(c, testConfig)` (`c` is config file in playground root) (non-resolved)
- `resolvedConfig`: resolved `rootConfig`

Maybe we should always set `configFile: false` in `options` and load the config manually?
Example:
```js
let config = await loadConfigFromFile(/* load from config file near the *.spec.ts */)
if (!config) {
  config = await loadConfigFromFile(/* load from config file in playground root */)
}
let resolvedConfig
const options = {
  configFile: false,
  plugins: [{ name: '', configResolved(config) { resolvedConfig = config } }]
  // omit
}
const mergedConfig = mergeConfig(options, config || {})

await build(mergedConfig)
await startStaticServer(mergedConfig)
```

Reply:
👍 It looks simpler